src/index.js

```diff
@@ -1,17 +1,117 @@
 import './style.css';
```

Comment:
# 💾 💾 💽 [OPTIONAL]   💽 💾 💾
**✔️ _Keep professionalism in your Repo's_** 
- ⏏︎ Kindly fill the information missing on your ``README`` file.